app/javascript/mastodon/selectors/settings.ts

```diff
@@ -25,6 +26,10 @@ export const selectSettingsNotificationsQuickFilterAdvanced = (
 ) =>
   state.settings.getIn(['notifications', 'quickFilter', 'advanced']) as boolean;
 
+export const selectSettingsGroupedNotifications = (state: RootState) =>
```

Comment:
The name is a bit unclear: what does `true` mean here?

Maybe `selectUseGroupedNotifications(state)` more clearly specify the intent?

Reply:
Renamed